src/commonComponents/cart/components/cartButton/CartButton.js

```diff
@@ -0,0 +1,46 @@
+// Prop-Types
```

Comment:
make this a separate commonComponent

Reply:
for cart we don't want a Button with 'Add' right? because it;s a;ready in the cart
see this component is actually needed?